ruff.toml

```diff
@@ -0,0 +1,76 @@
+# Enable rules.
+select = [
+    "A",
+    "C4",
+    "C90",
+    "COM",
```

Comment:
could you add a comment with that this means and/or link to Ruff docs for more context?

Reply:
Done